cl8-web/.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run build --if-present
```

Comment:
This should compile all of the Vue into plain HTML, CSS, and JS, which then has no tests run against it. Definitely a good step for deploying, but I don't think it is necessary for testing.